tests/pagination/pagination_test.php

```diff
@@ -58,8 +57,27 @@ public function setUp(): void
 			$request
 		);
 
+		$db = $this->getMockBuilder('\phpbb\db\driver\mysqli')
+			->disableOriginalConstructor()
+			->getMock();
+
 		$this->routing_helper = new \phpbb\routing\helper($this->config, $router, $symfony_request, $request, $filesystem, '', 'php');
-		$this->helper = new phpbb_mock_controller_helper($this->template, $this->user, $this->config, $symfony_request, $request, $this->routing_helper);
+		$this->helper = new phpbb_mock_controller_helper(
+			new \phpbb\auth\auth(),
+			new \phpbb\cache\driver\dummy(),
+			new \phpbb\cron\manager([], $this->routing_helper, '', 'php'),
+			$db,
+			new phpbb_mock_event_dispatcher(),
+			new \phpbb\language\language(new \phpbb\language\language_file_loader($phpbb_root_path, $phpEx)),
+			$this->template,
+			$this->user,
+			$this->config,
+			$symfony_request,
+			$request,
+			$this->routing_helper,
+			'',
```

Comment:
I understand it's not needed here but why not send the correct admin path anyway?